master/reference/public-cloud/digital-ocean.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Calico on Digital Ocean
+---
+
+Calico is designed to provide high performance massively scalable virtual
+networking for containers in public cloud environments such as Digital Ocean.
+
+## How to Run Calico in Digital Ocean
+
+Digital Ocean offers two networking options for droplets.
+Calico can be launched on either:
+
+- Public Interface
+- Private Interface [Optional]
```

Comment:
not sure I understand why there's an `[Optional]` tag here. You can have a Calico network completely on the Private interface, same goes for Public interface, `[Optional]` suggests Public one is mandatory but Private is not. WDYT?